### ISSUE TITLE
frontpage: Say exactly what OSMD is

### ISSUE THREAD

**Issue by sschmidTU** [opensheetmusicdisplay/opensheetmusicdisplay.github.io#12]
The front page of OSMD could be clearer about what OSMD is.
The banner subtitle says "The first JavaScript engine for MusicXML using VexFlow", but i actually didn't notice that when i looked at the front page for the first time after a while just now.
It doesn't say exactly that OSMD renders music xml files (sheet music) in the browser. And an engine could also be a modeling tool.

The "About OpenSheetMusicDisplay" text also doesn't make it clear. To be fair, it's very well written and catchy and does everything else well.

The first sentence of the first paragraph says:
"OpenSheetMusicDisplay is the missing link between MusicXML and VexFlow."

I would make it something like:
**"OpenSheetMusicDisplay renders sheet music in the Browser using MusicXML and VexFlow, creating the missing link between the two."**

less catchy, but clearer.

(ideally, i would like a less technical term than "render", because less technologically experienced people might not understand it, but on the other hand most visitors will)

This will also be clearer when we show a screenshot of OSMD along with the text.

**Comment by sebastianhaas** [comment]
> It doesn't say exactly that OSMD renders music xml files (sheet music) in the browser. And an engine could also be a modeling tool.

+1 for mentioning MusicXML, I guess that's also quite important for search engines? @bukaznik? what about SEO on that page? What's up with #4? Shouldn't we finally get this done and merge in?

> OpenSheetMusicDisplay renders sheet music in the Browser using MusicXML and VexFlow, creating the missing link between the two.

I'll leave that up to you guys in the office, I agree, however, that this is a technical page so I get the point of making clear what technology it actually uses and where. What about `Based on VexFlow, OpenSheetMusicDisplay renders sheet music in MusicXML format in your web browser.` Don't know if the missing link thing is really that important. I would google for MusicXML and VexFlow if I wanted something that connects these two.
> This will also be clearer when we show a screenshot of OSMD along with the text.

+100 this is such a no-brainer. I wonder about it every time I stumble across that page.


Thanks for bringing up stuff like that! <3

**Comment by phonicscore** [comment]
yes, yes let's do that :) @sschmidTU let's go through this next week in the office.

@sebastianhaas #4 won't be necessary anymore...remember when we went for lunch, I told you that I/we want to use wordpress again.

So if you type osmd.org in your browser in the future you will land at the wp-site with a big button directing to github (and from therelinking to the demo and all the dev stuff like class documentation etc). Content like blog posts, team page etc will be on wp (as we used to have it in the beginning). It's just more convenient for the biz people you know ;) And it's SEO friendly out of the box...
